app/models/article.rb

```diff
@@ -1,5 +1,8 @@
 class Article < ApplicationRecord
-  has_many :comments
+  include Visible
```

Comment:
What is done on this line?

Reply:
Is included the Visible module, this is done for re-use code making use of the concept "Concerns", these are a way to make large controllers or models easier to understand and manage